### PR TITLE
[Elm] - Mentoring note for Two Fer

### PR DESCRIPTION
**Pull request by leojpod**
Add the first mentoring note on the Elm-track

### CODE REVIEW

File: tracks/elm/exercises/two-fer/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Two Fer
```
Comment (SleeplessByte) [inline tracks/elm/exercises/two-fer/mentoring.md]: Please remove this

File: tracks/elm/exercises/two-fer/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+_Two Fer_ is the first core exercise after _Hello, World_. _Hello, World_ is auto-approved, so _Two Fer_ is the first exercise where someone will be blocked until a mentor approves their exercise, and also likely the first feedback they've ever gotten on their Go code.
+
+This is also going to be the most common exercise that we are asked to mentor, since some people will drop off after submitting just the one.
+
+The goal of the exercise is to get acquainted with Elm. This is often an excellent place to start introducing people to the tooling.
+
+## Reasonable solutions
```
Comment (SleeplessByte) [inline tracks/elm/exercises/two-fer/mentoring.md]: Use level-three heading: `### Reasonable solutions`

File: tracks/elm/exercises/two-fer/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+We don't have any clear guidelines around when to approve. Some people have been adding comments and approving straight away, others have been approving when the solution has a nice `Maybe.withDefault` in it. It might be a good time to introduce it if the students are not using it and also a good time to point them to package.elm-lang.org since they are probably going to spend some time on it if they stick around. A good solution might look like:
+
+```
```
Comment (SleeplessByte) [inline tracks/elm/exercises/two-fer/mentoring.md]: Annotate as elm for code highlight
```elm
module TwoFer exposing (twoFer)
twoFer : Maybe String -> String
twoFer name =
    "One for " ++ Maybe.withDefault "you" name ++ ", one for me."
```

File: tracks/elm/exercises/two-fer/mentoring.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+module TwoFer exposing (twoFer)
+
+
+twoFer : Maybe String -> String
+twoFer name =
+    "One for " ++ Maybe.withDefault "you" name ++ ", one for me."
+```
+
+## Common suggestions
```
Comment (SleeplessByte) [inline tracks/elm/exercises/two-fer/mentoring.md]: Use level-three heading: ### Common suggestions

**Comment by leojpod** [comment]
@SleeplessByte Thanks for the review, I'll deal with it in a moment. 
May I ask why removing the title and adding a level to the other subtitle?

**Comment by SleeplessByte** [comment]
Thank you!


It's because of the current styling on the website. If you check the FAQ for mentor notes it's says to use level three headings, because level two and one will just look wrong.


Consistency over anything :D

**Comment by SleeplessByte** [comment]
@leojpod thank you for the work! <3 

@exercism/elm merged this because _something_ is better than nothing.

**Comment by leojpod** [comment]
@SleeplessByte Cheers :) Love the justification :D

**Pull request merged**